security/storage/aws/public_buckets/README.md

```diff
@@ -1,15 +1,13 @@
-### AWS Open Buckets Policy Template
-
-**What it does**
-This Policy Template will check your account for amazon s3 buckets with public permission. It takes several parameters: 
-- `Email address to send escalation emails to` - Email to alert when it finds s3 buckets that meet the criteria
-
-## Supported Clouds
-The following clouds are supported: 
-- AWS
-- Azure
-- Google 
-
-**Cost**
-
-This Policy Template does not launch any instances, and so does not incur any cloud costs.
\ No newline at end of file
+### AWS Open Buckets Policy Template
+
+**What it does**
+This Policy Template will check your account for amazon s3 buckets with public permission. It takes several parameters: 
```

Comment:
looks like it only takes one param, so remove several parameters and replace with one parameter.